units_cpp/tech.cc

```diff
@@ -1,75 +1,95 @@
 #include <iostream>
 #include <sstream>
-#include "tech.h"
+
 #include "json.h"
+#include "tech.h"
+
+
+Tech::Tech():
+		red(0),
+		green(0),
```

Comment:
If it fits on one line, there's no need to make it five (unless there is some very advantageous gain in code alignment or readability; not the case here).